Formula/f/fcft.rb

```diff
@@ -0,0 +1,52 @@
+class Fcft < Formula
+  desc "Simple library for font loading and glyph rasterization"
+  homepage "https://codeberg.org/dnkl/fcft"
+  url "https://codeberg.org/dnkl/fcft/archive/3.1.8.tar.gz"
+  sha256 "f48c793f354b8be95477e475dde7c98ac9d2628c52ecb42029dc6d20b52d787c"
+  license "MIT"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "scdoc" => :build
+  depends_on "tllist" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "harfbuzz"
+  depends_on :linux
```

Comment:
Can't see where this requires Linux -- do you know where the source relies on this?

Reply:
The failed MacOSX CI test complained about the missing `threads.h` header, which is included in `fcft.h` and is part of the [Linux headers](https://github.com/torvalds/linux/blob/master/include/linux/threads.h). I did not find any direct mention to the constants defined in that file, though.

Reply:
I see. That sounds something that can be easily fixed upstream. I remember @cho-m handling this issue for another formula recently.

Reply:
Ok, I opened an [issue](https://codeberg.org/dnkl/fcft/issues/67) on `fcft`'s repo to see if we can clarify this.

Reply:
Ok, according to [fcft's author](https://codeberg.org/dnkl/fcft/issues/67#issuecomment-2354617) this is just a matter of using a compiler and libc that supports `threads.h`, so I changed the dependencies on mac to use both gcc and glibc. I don't know if this is the right way to do it, so please let me know how to proceed, whenever it's convenient for you.

Reply:
`glibc` isn't available on macOS. I don't think you can even build it. Same thing with `musl` (not available as formula and won't build on macOS).

So adding a dependency on it is identical to `depends_on :linux` https://github.com/Homebrew/homebrew-core/actions/runs/11194734088/job/31121620578?pr=192887#step:3:30
```
==> SKIPPED fcft
Warning: fcft has dependencies without compatible bottles:
  glibc
```

Reply:
Thanks for the clarification. What should I do, then?

Reply:
We can probably accept it as `depends_on :linux` with a comment to upstream issue.

I don't think there is a C library on macOS that supports C11 threads. macOS support will require either a fallback to POSIX threads or a compatibility layer like `gnulib` (but this can impact license) or a 3rd-party library.

Reply:
So, unless I'm misunderstanding, the most viable option for now is to go with `depends_on :linux` with a comment to the upstream issue. Is that correct?

Reply:
I added back the dependency on `:linux` and removed the MacOSX-specific code again.